vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,157 @@
+
+```bash
+..\vcpkg\vcpkg new --name helloworld --version 1.0
+```
+
+>[!NOTE]
+>Ensure the project directory name matches exactly with the name you give to the `vcpkg new` command. This ensures the manifest files land in the correct directory.
+
+As mentioned above, the `vcpkg new` command will add a `vcpkg.json` file and a `vcpkg-configuration.json` file to your projects directory. The `vcpkg.json` file should look like this:
+
+```json
+{
+    "name": "helloworld",
+    "version": "1.0"
+}
+```
+
+This `vcpkg.json` file serves as the foundation for your vcpkg-based C++ project. It identifies your project by name and version, setting the stage for additional attributes and dependencies that might be added as your project grows.
+
+To learn more about `vcpkg.json`, check out our reference [documentation](..\reference\vcpkg-json.md).
+
+The `vcpkg-configuration.json` file serves a different purpose. This file essentially allows you to specify which registry vcpkg should use to look for packages (both by default and additionally). It should look like this:
+
+```json
+{
+    "default-registry": {
+        "kind": "git",
+        "baseline": "b98e11a16474488633d8a1ab3e160da1f94b98e2",
+        "repository": "https://github.com/microsoft/vcpkg"
+},
+    "registries": [
+        {
+        "kind": "artifact",
+        "location": "https://github.com/microsoft/vcpkg-ce-catalog/archive/refs/heads/main.zip",
+        "name": "microsoft"
+        }
+    ]
+}
+```
+
+There's a lot going on here, but lets break it down line-by-line starting with the `default-registry` JSON object:
+
+As the name suggests, the `"default-registry"` specifies the default location vcpkg should use to look for packages. The `"default-registry":{` key specifies the default package-registry settings containing details about the registry.
+
+* `"kind": "git",`: Specifies the type of registry. In this case, it's a Git repository.
+* `"baseline": "b98e11a16474488633d8a1ab3e160da1f94b98e2",`: Represents a set of versions which are likely to work together.
+* "`"repository": "https://github.com/microsoft/vcpkg"`: Is the URL of the Git repository to be used as the default registry.
+
+This tutorial only uses the default registry, but know that you can specify additional registries by adding them to the `registries` array. Note that different kinds of registries will have different attributes. In this case, the `registries` array contains one additional registry that's different from the default. Let's break it down:
+
+* `"kind": "artifact,"`: Specifies the type of this additional registry. In this example, it's an "artifact" type.
+* `"location": https://github.com/microsoft/vcpkg-ce-catalog/archive/refs/heads/main.zip",`: The URL from which this artifact registry can be fetched.
+* `"name": "microsoft"`: Specifies the name for this additional registry, which is "microsoft".
```

Comment:
```suggestion
```

Reply:
This will be covered by an in-depth tutorial

Reply:
@AugP, suggested I give a detailed explanation for each line in the JSON snippet. I wasn't aware there would be more in-depth tutorials?

Reply:
@vicroms in this case this is the tutorial. If you mean there will be how-to guides as well, I would say those would be less wordy since they would focus on getting the user to complete the task in a practical way and would be less concerned with teaching the user about exactly how everything works. 

I think the Get started content needs to accomplish two things:
1. Provide only the minimal information the user needs to know to complete the tutorial (avoid overwhelming with information). Give the user a simple task and keep them laser-focused on that task. 
2. For anything that is referenced in the tutorial (part of that minimal information), describe everything that is referenced thoroughly, so the user is sure of what everything does.

Reply:
@AugP, I think your points are a bit contradictory. Given those two goals, I'm not sure how much information is _minimal_ but also describes "everything that is referenced thoroughly"

Reply:
Actually, I think I misunderstood what Victor was saying. There will indeed be a separate tutorial on creating and publishing packages, which can go into the vcpkg-configuration.json topic. Yes, it can be removed from this tutorial if the user doesn't need to actually edit this file. 

The two points are basically saying that a tutorial should have a specific, learning-oriented task for the user to accomplish. We should not go off-script to talk about other topics (even if they are loosely related), but for the topics we do talk about (the ones for the task at hand), each concept should be described without the user having to look at other documentation to understand what the tutorial is talking about.